client/my-sites/plan-features/index.jsx

```diff
@@ -37,68 +35,179 @@ import { planItem as getCartItemForPlan } from 'lib/cart-values/cart-items';
 
 class PlanFeatures extends Component {
 	render() {
-		if ( this.props.isPlaceholder ) {
-			return <PlanFeaturesPlaceholder />;
+		const { isPlaceholder } = this.props;
+
+		if ( isPlaceholder ) {
+			//TODO: update placeholder
+			return null;
 		}
 
-		const {
-			available,
-			currencyCode,
-			planName,
-			rawPrice,
-			discountPrice,
-			popular,
-			current,
-			planConstantObj,
-			features,
-			onUpgradeClick
-		} = this.props;
-
-		const classes = classNames( 'plan-features', {
-			'is-popular': popular
+		return (
+			<div className="plan-features">
+				<table className="plan-features__table">
+					<tbody>
+						<tr>
+							{ this.renderPlanHeaders() }
+						</tr>
+						<tr>
+							{ this.renderPlanDescriptions() }
+						</tr>
+						<tr>
+							{ this.renderTopButtons() }
+						</tr>
+						{ this.renderPlanFeatureLists() }
```

Comment:
In this case `renderPlanFeatureLists` will also produce a number of <tr> elements. I agree that it does look a bit odd.